latest/examples/subspace/subspaceMultidim.json

```
@@ -0,0 +1,64 @@
+{
+    "coordinateSystems" : [
+        { "name " : "in", "axes" : [ {"name" : "0", "name" : "1", "name": "2", "name": "3", "name": "4" }] },
```

The axes list has a single dict with multiple "name" keys? Is is supposed to be `[{"name" : "0"}, {"name" : "1"}..]` ?

---

```suggestion
        { "name ": "in", "axes": [{"name" : "0"}, {"name" : "1"}, {"name": "2"}, {"name": "3"}, {"name": "4"}] },
```

@bogovicj, should changes to the examples happen here or in your PR?

I haven't modified these from your branch, but also could add tests for the subspace directory.

---

https://github.com/bogovicj/ngff/blob/coord-transforms-schemas/latest/examples/subspace/subspaceMultidim.json#L3-L20